usvg/src/convert/text/convert.rs

```diff
@@ -317,9 +318,30 @@ fn resolve_text_flow(
         node.resolve_length(AId::StartOffset, state, 0.0)
     };
 
+    let node_transform = path_node.attribute::<Transform>(AId::Transform).unwrap_or_default();
+
+    // Get the absolute transform by looking at all parent elements
+    // Unfortunately, we can't use NodeExt::abs_transform since we're still 
+    // only dealing with svgtree::node
+    let mut transform_stack = vec![node_transform];
+
+    let mut current_node = path_node;
```

Comment:
Are you sure that we need an absolute transform? From my brief tests it look like only the element transform is used.

Reply:
Whoops - you are right! That wasn't my intuition, but the spec is pretty unambigouous here:

> The path data coordinates within the referenced ‘path’ element are assumed to be in the same coordinate system as the current ‘text’ element, **not in the coordinate system where the ‘path’ element is defined.** The ‘transform’ attribute on the referenced ‘path’ element represents a supplemental transformation relative to the current user coordinate system for the current ‘text’ element, including any adjustments to the current user coordinate system due to a possible ‘transform’ attribute on the current ‘text’ element.

Now I know. Ah well...
Thanks for the feedback!

Reply:
At least that explains why some of my svgs I was creating look so bogus :)